### ISSUE TITLE
bad release of amazonka-1.4.4.2

### ISSUE THREAD

**Issue by gregwebs** [brendanhay/amazonka#329]
I think 10486b8fb0e17a071b4e2d81fe201164e0dcdb84 is what is on hackage. When I use, this, it doesn't fix #322 

If I use c695a918c8e52400063c2810d5da4d47de6da628 then the issue is fixed.

**Comment by screamish** [comment]
I am seeing the same, the bug persists for me when I reference 1.4.4.2 in my stack.yml like so:

```
extra-deps:
- amazonka-1.4.4.2
```

**Comment by gregwebs** [comment]
This works well for me

``` yaml
packages
  - .
  - location:
      git: https://github.com/brendanhay/amazonka.git
      commit: c695a918c8e52400063c2810d5da4d47de6da628
    subdirs:
      - amazonka
    extra-dep: true
```

**Comment by brendanhay** [comment]
I profusely apologise for this. There are some upstream service definition updates which I'll integrate and test before releasing something like `1.5`, to also include c695a91 and beyond.

**Comment by DanBurton** [comment]
+1 Also noticed this. `stack unpack amazonka-test-1.4.4.2`, inspect `src/Test/AWS/Orphans.hs`, matches code prior to #324.

**Comment by brendanhay** [comment]
This should now be fixed per release [1.4.5](http://hackage.haskell.org/package/amazonka-1.4.5).
